invenio_rdm_migrator/transform/transactions.py

```diff
@@ -5,20 +5,33 @@
 # Invenio-RDM-Migrator is free software; you can redistribute it and/or modify
 # it under the terms of the MIT License; see LICENSE file for more details.
 
-"""Invenio RDM migration transaction group transform."""
+"""Invenio RDM migration transaction transform."""
+
+from abc import ABC
 
 from ..actions.base import TransformAction
 from .base import Transform
 
 
-class Tx(Transform):
-    """Transform a transaction group and its items."""
+class BaseTxTransform(Transform, ABC):
+    """Transform a transaction."""
+
+    actions: list[TransformAction] = []
 
-    def _transform(self, entry):
+    def _detect_action(self, tx):
```

Comment:
Does it have a diff name on purpose? what is the difference in "meaning" between match and detect? (wondering if there is some specific responsability the name is implying...)

Reply:
Not really... it's more that we detect the action why trying a bunch of matches... so I couldn't find a better abstraction. In a similar fashion, if you had a bunch of rules/regexes and wanted to run them all through to see which one matches, what would be the verb/action?